pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -64,6 +64,15 @@ var validInterfaceModels = map[string]*struct{}{"e1000": nil, "e1000e": nil, "ne
 var validIOThreadsPolicies = []v1.IOThreadsPolicy{v1.IOThreadsPolicyShared, v1.IOThreadsPolicyAuto}
 var validCPUFeaturePolicies = map[string]*struct{}{"": nil, "force": nil, "require": nil, "optional": nil, "disable": nil, "forbid": nil}
 
+var filteredVmiKubevirtLabels = map[string]bool{
```

Comment:
Maybe call it `restriectedVmiLabels` instead?

Reply:
This can be a slice. As Map doesn't add additional value IMO

Reply:
@danielBelenky I've rnamed it. @xpivarc AFAIK map lookup is better then iterating over a slice